include/ensmallen_bits/sdp/primal_dual_impl.hpp

```diff
@@ -227,46 +186,8 @@ SolveKKTSystem(const SparseConstraintType& aSparse,
   dsZ = rd - subTerm;
 }
 
-template<typename DeprecatedSDPType>
-ens_deprecated
-double PrimalDualSolver<DeprecatedSDPType>::Optimize(
-    arma::mat& coordinates)
-{
-  // Use the internally-held initial parameters.
-  coordinates = initialX;
-  arma::mat ySparse = initialYSparse;
-  arma::mat yDense = initialYDense;
-  arma::mat Z = initialZ;
-  return Optimize<DeprecatedSDPType, arma::mat>(deprecatedSDP, coordinates,
-      ySparse, yDense, Z);
-}
-
-template<typename DeprecatedSDPType>
-ens_deprecated
-double PrimalDualSolver<DeprecatedSDPType>::Optimize(
-    arma::mat& coordinates,
-    arma::vec& ySparse,
-    arma::vec& yDense,
-    arma::mat& z)
-{
-  // Initialize internally then call the other overload.
-  coordinates = initialX;
-  arma::mat ySparseMat = initialYSparse;
-  arma::mat yDenseMat = initialYDense;
-  z = initialZ;
-
-  const double result = Optimize<DeprecatedSDPType, arma::mat>(deprecatedSDP,
-      coordinates, ySparseMat, yDenseMat, z);
-
-  ySparse = ySparseMat.col(0);
-  yDense = yDenseMat.col(0);
-
-  return result;
-}
-
-template<typename DeprecatedSDPType>
 template<typename SDPType, typename MatType, typename... CallbackTypes>
-typename MatType::elem_type PrimalDualSolver<DeprecatedSDPType>::Optimize(
+typename MatType::elem_type PrimalDualSolver::Optimize(
     const SDPType& sdp,
```

Comment:
You can move this to the line above and align the parameters as well

Reply:
Can't align the parameters and still meet the <80 character requirement (see line 192 - `CallbackTypes&&...` would go over 80)

Reply:
You're right, I hadn't noticed. Sorry about that.